api/src/main/java/org/openmrs/module/fhir2/api/FhirPatientService.java

```diff
@@ -9,13 +9,15 @@
  */
 package org.openmrs.module.fhir2.api;
 
-import java.util.Collection;
```

Comment:
Please make sure your IDE is configured according to the [OpenMRS standard](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE). In particular, `java` and `javax` import should come before other imports (except static).